### DISCUSSION TITLE
Can't type on remote desktop when using Router

### DISCUSSION THREAD

**Discussion by eagle856** [Ylianst/MeshCentral#4164] [Q&A]
As the title says, if I initiate a remote session to a Windows 10 computer using MeshCentral Router, I'm unable to type anything (including login credentials). Everything works fine using the web version. I've tried setting the option to use remote  keyboard map (in Router and the web version), but it had no affect. Any help would be greatly appreciated!

**Comment by si458** [comment]
Plz can you Try the latest version from 1.0.54 and see if this issue still exists?

**Comment by Ylianst** [comment]
This was probably answered in the issues section, but if this happens, make sure to turn off the "Enhanced keyboard capture" options in the MeshCentral Router settings:

![image](https://user-images.githubusercontent.com/1319013/180137993-4d0292b9-de11-4b3c-9e77-c50db49c51de.png)

I think in this case, that was the problem.
